doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -887,7 +887,9 @@ Common Application Framework
 Debug libraries
 ---------------
 
-|no_changes_yet_note|
+* :ref:`cpu_load` library:
```

Comment:
Nit: I would expect this changelog update in the same commit as the change itself. Is this just temporary and will be squashed later?